### ISSUE TITLE
Layout crash when all data points are zero

### ISSUE THREAD

**Issue by ChristianOrgler** [willdale/SwiftUICharts#114]
**Pre-Info**
- I've tried the chart already with values, works perfectly
- Runtime: iOS 15 beta 8

**Usage-Case**
- Showing seven (weekdays) data points which have the value `0` (I don't want to use the no-data label)

**Crash**
```
*** Terminating app due to uncaught exception 'CALayerInvalidGeometry', reason: 'CALayer position contains NaN: [nan nan].

Layer: <_TtC7SwiftUIP33_F176A6CF4451B27508D54E2BEAEBFD5415PaintShapeLayer:0x6000032cefd0; position = CGPoint (0 0); bounds = CGRect (0 0; 0 0); delegate = <_TtC7SwiftUIP33_A34643117F00277B93DEBAB70EC0697122_UIShapeHitTestingView: 0x12ca09c20; frame = (-8.98847e+307 -8.98847e+307; 1.79769e+308 1.79769e+308); transform = [nan, nan, nan, nan, 0, 0]; anchorPoint = (0, 0); autoresizesSubviews = NO; layer = <_TtC7SwiftUIP33_F176A6CF4451B27508D54E2BEAEBFD5415PaintShapeLayer: 0x6000032cefd0>>; allowsEdgeAntialiasing = YES; allowsGroupOpacity = YES; anchorPoint = CGPoint (0 0); contentsScale = 2; transform = CATransform3D (nan nan 0 0; nan nan 0 0; 0 0 1 0; 0 0 0 1)>'
terminating with uncaught exception of type NSException
```

**Comment by willdale** [comment]
Hi, what chart type is it?

**Comment by ChristianOrgler** [comment]
Oh sorry, BarChart

**Comment by willdale** [comment]
No worries, I'll have a look.

**Comment by willdale** [comment]
Fixed in v2.9.2.

Thanks
Will

**Comment by ChristianOrgler** [comment]
Thanks for the fast fix!
